controllers/disruption_controller.go

```diff
@@ -652,29 +648,27 @@ func (r *DisruptionReconciler) selectTargets(instance *chaosv1beta1.Disruption)
 
 // getChaosPods returns chaos pods owned by the given instance and having the given labels
 func (r *DisruptionReconciler) getChaosPods(instance *chaosv1beta1.Disruption, ls labels.Set) ([]corev1.Pod, error) {
-	ownedPods := make([]corev1.Pod, 0)
 	pods := &corev1.PodList{}
+	var err error
+	if ls == nil {
+		ls = make(map[string]string)
+	}
+	// If this isn't a valid label, we will instead list ALL pods
```

Comment:
We can be guaranteed that Namespace and Name are both legal substrings in a label value, but previously I was joining them with "/", which is _not_ okay in a label value.